### PR TITLE
Updated contributor list

### PR DESCRIPTION
**Pull request by ysharma1126**

### CODE REVIEW

**Comment by ysharma1126** [comment]
I'm not sure of the cited errors in the build, all I did was update the Readme.

File: README.md [hunk 1]
```diff
@@ -179,18 +179,29 @@ Ryan Sheatsley (Pennsylvania State University).
...
-* Alexey Kurakin (Google Brain)
+* Zhi Li
 * Ryan Sheatsley (Pennsylvania State University)
 * Abhibhav Garg (IIT Delhi)
-* Yen-Chen Lin (National Tsing Hua University)
+* Willi Gierke (Hasso Plattner Institute)
+* Yinpeng Dong (Tsinghua University)
+* Jonas Rauber (IMPRS)
 * Paul Hendricks
+* David Berthelot
```
Comment (goodfeli) [inline README.md]: David is at Google Brain

File: README.md [hunk 2]
```diff
@@ -179,18 +179,29 @@ Ryan Sheatsley (Pennsylvania State University).
...
+* Shuolongbj
```
Comment (goodfeli) [inline README.md]: Do we know any more about this contributor? Can you try to contact them?

**Comment by goodfeli** [comment]
It might make sense to make this the last thing we do after getting the release ready, since the PRs we're merging while getting the release ready are changing the author order / list

**Comment by goodfeli** [comment]
I'll send a PR to fix the build

**Comment by ysharma1126** [comment]
I don't know anything more about the contributor, no contact information on
the Github.

I'll send my final contributor list in a pull request once the bugs have
been fixed then.

**Comment by goodfeli** [comment]
I think this is because we use `pip install keras` with no version specification on Travis. A new release of Keras came out a few days ago and it looks like the new release isn't compatible with the older versions of TensorFlow that we use on Travis. This PR should fix it: https://github.com/tensorflow/cleverhans/pull/399

**Comment by ysharma1126** [comment]
The Keras tutorial still fails. Can you please tell Travis to do another build?

**Pull request merged**